README.md

```diff
@@ -67,7 +67,7 @@
 
 ## 📹 Video Description
 
-- [Video Description Link](https://www.loom.com/share/bb51379e56854720a0178d1cf8c4d2c5?sid=e109121e-20f7-49f7-b0e0-9bfd8f54962d)
+- [Video Description](https://www.loom.com/share/bb51379e56854720a0178d1cf8c4d2c5?sid=e109121e-20f7-49f7-b0e0-9bfd8f54962d)
 
```

Comment:
* I get noticed this bug when I try to submit an empty or half-filled form the form submits anyways with an error kindly look into it 

https://github.com/shaaibu7/Budget-app/assets/52497246/e6b312b4-fe2e-48cc-96a7-18ff53d09711

Reply:
This change has been done well